File: Find rectangle with corners as 1.java
class Solution {
    public boolean ValidCorner(int mat[][]) {
        // Code here
        int c=0;
        if(mat[0].length>1&&mat.length>1)
        for(int i=0;i<mat.length;i++){
            for(int j=0;j<mat[0].length;j++){
                if(mat[i][j]==1){
                    for(int k=j+1;k<mat[0].length;k++){
                        if(mat[i][k]==1){
                            for(int s=i+1;s<mat.length;s++){
                                if(mat[s][k]==1&&mat[s][j]==1)
                                return true;
                            }
                        }
                    }
                }
            }
        }
        return false;
    }
}

File: Find rectangle with corners as 1.py
class Solution:    
    def ValidCorner(self,mat): 
        # Code here 
        c = 0
        if len(mat) > 1 and len(mat[0]) > 1:
            for i in range(len(mat)):
                for j in range(len(mat[0])):
                    if mat[i][j] == 1:
                        for k in range(j + 1, len(mat[0])):
                            if mat[i][k] == 1:
                                for s in range(i + 1, len(mat)):
                                    if mat[s][k] == 1 and mat[s][j] == 1:
                                        return True
        return False
